youtube_dl/YoutubeDL.py

```diff
@@ -1190,9 +1190,9 @@ def _calc_headers(self, info_dict):
         if add_headers:
             res.update(add_headers)
 
-        cookies = self._calc_cookies(info_dict)
-        if cookies:
-            res['Cookie'] = cookies
+        #cookies = self._calc_cookies(info_dict)
+        #if cookies:
+        #    res['Cookie'] = cookies
```

Comment:
i will return it back later.
self._calc_cookies depend on url attribute(url attribute is not available directly in the info dict if the parts are present) so i commented it temporaly to make this change to work and create this pull request to see if the idea is good enough than i will continue to work in it,if it's not than i will close it.